sdk/web-pubsub/web-pubsub-client/test/client.lifetime.spec.ts

```diff
@@ -81,16 +81,17 @@ describe("WebPubSubClient", function () {
         const client = new WebPubSubClient("wss://service.com", {
           messageRetryOptions: { retryDelayInMs: 10 } as WebPubSubRetryOptions,
         } as WebPubSubClientOptions);
-        const mock = sinon.mock(client);
-        mock
-          .expects("_sendMessage")
-          .exactly(4)
-          .withArgs(expectMessage)
-          .callsFake((_) => Promise.reject());
+
+        const mock = vi
+          .spyOn(client as any, "_sendMessage")
+          .mockImplementation(() => Promise.reject());
+
         try {
           await actualMethod(client);
         } catch {}
-        mock.verify();
+
+        expect(mock).toHaveBeenCalledWith(expectMessage, undefined);
```

Comment:
Why do we need `undefined` here now? vitest expect requires specifying all parameters even for undefined?

Reply:
Correct, all parameters must be defined including `undefined`